source/developer-guides/debug.rst

```diff
@@ -0,0 +1,112 @@
+.. _debug:
+
+Source Level Debugging on Serial Port
+--------------
+
+This explains how to perform basic debugging operation with Intel UDK Debugger on SBL over serial port.
+
+**Prerequisite**
```

Comment:
Instead of using Bold text, we should use sub-section headers here. Like: `### Prerequisites`. Repeated occurrences below.